native_client/README.rst

```diff
@@ -197,3 +198,20 @@ And (ARM64):
 
    cd ../DeepSpeech/native_client
    $ANDROID_NDK_HOME/ndk-build APP_PLATFORM=android-21 APP_BUILD_SCRIPT=$(pwd)/Android.mk NDK_PROJECT_PATH=$(pwd) APP_STL=c++_shared TFDIR=$(pwd)/../tensorflow/ TARGET_ARCH_ABI=arm64-v8a
+
+Delegation API
+^^^^^^^^^^^^^^
+
+TensorFlow Lite supports Delegate API to offload some computation from the main CPU. Please refer to `TensorFlow's documentation <https://www.tensorflow.org/lite/performance/delegates>`_ for details.
+
+To ease with experimentations, we have enabled some of those delegations on our Android builds:
+* GPU, to leverage OpenGL capabilities
+* NNAPI, the Android API to leverage GPU / DSP / NPU
+* Hexagon, the Qualcomm-specific DSP
+
+This is highly experimental:
+* Requires passing environment variable ``DS_TFLITE_DELEGATE`` with values of ``gpu``, ``nnapi`` or ``hexagon`` (only one at a time)
+* Might require exported model changes (some Op might not be supported)
+* We can't guarantee it will work, nor it will be faster than default implementation
+
+Feedback on improving this is welcome: how it is exposed in the API, how much performance speed do you get, how much changes does a delegate requires to the model, etc.
```

Comment:
```suggestion
Feedback on improving this is welcome: how it could be exposed in the API, performance gains you get in your applications, how you had to change to model to make it work with a delegate, etc.
```

Reply:
We should also include `SUPPORT.rst` in ReadTheDocs, and link to it in this paragraph.